.github/workflows/prod-ci-cd.yml

```
@@ -0,0 +1,99 @@
+env:
+  REGISTRY: docker.io
+  IMAGE_NAME: ${{ secrets.DOCKER_USERNAME }}/wonderwander
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    timeout-minutes: 25
+    outputs:
+      image_tag: ${{ steps.meta.outputs.tags }}
+      version: ${{ steps.meta.outputs.version }}
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)